src/main/java/ca/etsmtl/leakageanalysisplugin/services/HttpClientLeakageService.java

```diff
@@ -17,24 +19,22 @@
 
 import static ca.etsmtl.leakageanalysisplugin.util.AnalysisUtil.isFileSupported;
 
-@Service(Service.Level.PROJECT)
 public final class HttpClientLeakageService implements LeakageService {
+    public static final MediaType ANALYZE_MEDIATYPE = MediaType.parse("application/octet-stream");
     private static final String BASE_URL = "http://localhost:5000";
-    private static final Long timeout = 120L;
-
+    private static final Long TIMEOUT = 120L;
     private final OkHttpClient client;
 
     public HttpClientLeakageService() {
         client = new OkHttpClient().newBuilder()
                 //.callTimeout(timeout, TimeUnit.SECONDS)
-                .readTimeout(timeout, TimeUnit.SECONDS).build();
+                .readTimeout(TIMEOUT, TimeUnit.SECONDS).build();
     }
 
     public AnalysisResult analyze(String filePath) {
```

Comment:
I think we should merge the function `analyze` and the function `analyzeFile`. Keeping them both now that the calls are merged server side is redundant IMO.

Reply:
Good idea